### PR TITLE
#767 copy username in chat

### PR DESCRIPTION
**Pull request by Geosearchef**
capture

### CODE REVIEW

**Review by Brutus5000 — DISMISSED** [review]

File: src/main/resources/i18n/messages_de_DE.properties [hunk 1]
```diff
@@ -65,6 +65,7 @@ chat.category.chatOnly=Nur Chat
 chat.userContext.noAvatar=Kein Avatar
 chat.userContext.userInfo=Zeige Benutzerinformationen
 chat.userContext.privateMessage=Private Nachricht
+chat.userContext.copyUsername=Nutzernamen kopieren
```
Comment (micheljung) [inline src/main/resources/i18n/messages_de_DE.properties]: `Kopiere Benutzernamen`
(Konsistent "Benutzername" und Imperativ verwenden)

**Review by micheljung — CHANGES_REQUESTED** [review]

**Comment by micheljung** [comment]
Please change the commit message to use `Fixes #767`, "finishes" [is not a valid keyword](https://help.github.com/articles/closing-issues-using-keywords/)

**Comment by Geosearchef** [comment]
encoding/text fixed, didn't care about the keyword as it diddn't work last time I tried to use one on this repo.

**Review by 1-alex98 — CHANGES_REQUESTED** [review]
Change encoding in seperate PR and rename commit messages

**Comment by Geosearchef** [comment]
done

**Review by micheljung — APPROVED** [review]

File: src/main/java/com/faforever/client/chat/ChatUserContextMenuController.java [hunk 2]
```diff
@@ -234,6 +236,12 @@ public void onSendPrivateMessage() {
     eventBus.post(new InitiatePrivateChatEvent(player.getUsername()));
   }
 
+  public void onCopyUsername() {
+    ClipboardContent clipboardContent = new ClipboardContent();
+    clipboardContent.putString(player.getUsername());
+    Clipboard.getSystemClipboard().setContent(clipboardContent);
+  }
```
Comment (micheljung) [inline src/main/java/com/faforever/client/chat/ChatUserContextMenuController.java:243]: After merging I realized, it'd be nice if there was a unit test for this.

**Pull request merged**